cosv-backend/src/main/kotlin/com/saveourtool/cosv/backend/CosvApplication.kt

```diff
@@ -9,9 +9,18 @@ import org.springframework.boot.context.properties.EnableConfigurationProperties
 import org.springframework.context.annotation.Import
 
 /**
- * An entrypoint for spring for save-cosv
+ * An entrypoint for spring for cosv-backend
  */
-@SpringBootApplication(scanBasePackages = ["com.saveourtool.save.utils", "com.saveourtool.save"])
+@SpringBootApplication(scanBasePackages = [
+    "com.saveourtool.save.configs",
```

Comment:
now I think we can have only two packages: `com.saveourtool.cosv` and `com.saveourtool.save`

Reply:
Yes. Changed it